### ISSUE TITLE
support mobile platform

### ISSUE THREAD

**Issue by fastfading** [Windblade-GR01/Ikemen-GO#224]
several years ago , I port original   https://osdn.net/users/supersuehiro/pf/ikemen_go/   to  mobile platform .
however it has performance issue then.   
its engine has performance issue.  update and draw should run separately.
many years past. seems you have done a lot .   
even import hajimehoshi lib,   
hajimehoshi/ebiten  lib is very good to support mobile.
have you consider to port this game to mobile  ios and android?

**Comment by Orochikyocr** [comment]
At least I saw one dev saying he has an interest in porting it to Android. Go has support for Android and IOS but now the problem is some broken audio dependencies.

**Comment by Adnan1091** [comment]
They use openAL.I linked to some projects which uses openAL but also has a mobile port like supertuxkart.I didn't get a response though.

**Comment by Windblade-GR01** [comment]
The main problem is GLFW.
It has no mobile support as now.

**Comment by fastfading** [comment]
GLFW could be replaced by gomobile  framework


Gacel <notifications@github.com> 于2021年2月22日周一 下午1:18写道：

**Comment by Adnan1091** [comment]
Yes,it should be replaced.Mobile port would make this application more famous.
